psh/sysexec/sysexec.c

```diff
@@ -34,7 +37,11 @@ static void psh_sysexecinfo(void)
 
 static void psh_sysexecUsage(void)
 {
-	fputs("Usage: sysexec [-m datamap] [-M codemap] progname [args]...\n", stderr);
+	fputs("Usage: sysexec [OPTIONS] progname [args]...\nOptions:\n", stderr);
+	fputs("\t-m datamap   select data memory map\n", stderr);
+	fputs("\t-M codemap   select code memory map\n", stderr);
+	fputs("\t-d           daemonize\n", stderr);
+	fputs("\t-s           do not close stdin on daemonization\n", stderr);
```

Comment:
Instead of five calls, just one (binary is shorter by 72bytes if one fputs is used)
```suggestion
	fputs("Usage: sysexec [OPTIONS] progname [args]...\n"
		"Options:\n"
		"\t-m datamap   select data memory map\n"
		"\t-M codemap   select code memory map\n"
		"\t-d           daemonize\n"
		"\t-s           do not close stdin on daemonization\n",
		stderr);
```